src/test/java/org/example/BaseTests.java

```diff
@@ -88,9 +110,27 @@ public void recordFailure(ITestResult result) {
 			} catch (IOException e) {
 				e.printStackTrace();
 			}
+			jiraService.createIssue("Test failed: " + result.getName());
+		}
+	}
+
+	@AfterMethod
+	public void afterMethod(ITestResult result) {
+		if (result.getStatus() == ITestResult.SUCCESS) {
+			passed++;
+		} else if (result.getStatus() == ITestResult.FAILURE) {
+			failed++;
+		} else if (result.getStatus() == ITestResult.SKIP) {
+			skipped++;
 		}
 	}
 
+	@AfterClass
+	public void afterClass() {
+		String message = String.format("Tests completed. %d passed, %d failed, %d skipped.", passed, failed, skipped);
+		slackNotifier.postNotification(message);
```

Comment:
you probably need to reset the variables failed, passed, skipped. As if you have several classes with tests, then the last class runned will provide the result amount of all the classes! if you want clear result you need to reset the variables to '0' after the notification with result sent.

Reply:
Got It Ivan!
Variables were reset at the end of the method!